src/platform/site-wide/sass/modules/_m-responsive-table.scss

```diff
@@ -0,0 +1,17 @@
+@media screen and (max-width: $medium-screen) {
+  table.responsive {
+    thead {
+      position: absolute;
+      clip: rect(0, 0, 0, 0);
+    }
+
+    tr {
+      display: block;
+      padding: 0.35em;
+    }
+
+    td {
+      display: block;
+    }
+  }
+}
```

Comment:
I'm confused on the relationship between this and src/applications/disability-benefits/view-payments/sass/responsive-table.scss?

Reply:
the scss file was called `src/applications/disability-benefits/view-payments/sass/ResponsiveTable.scss` but i renamed it.

This file contained the necessary css that is being moved to `_m-responsive-table.scss` to create a responsive table css class to resolve https://github.com/department-of-veterans-affairs/va.gov-team/issues/11055

From https://github.com/department-of-veterans-affairs/vets-design-system-documentation/issues/203#issuecomment-679138608 it seemed that the css for a responsive tables was more useful as available for platform wide use instead of a ResponsiveTable component.

Reply:
https://github.com/department-of-veterans-affairs/vets-design-system-documentation/issues/203 mentioned the work for https://github.com/department-of-veterans-affairs/vets-website/blob/master/src/applications/disability-benefits/view-payments/components/responsive-table/ResponsiveTable.jsx as being an example for creating ResponsiveTables so pulled the responsive css attributes used for it into platform

Reply:
Do you think it makes sense to pull the whole SCSS file into platform, or are there problems with that?

Reply:
The css in `_m-responsive-table` is the minimum css needed to cause 

![image](https://user-images.githubusercontent.com/1094999/93794805-94857900-fc06-11ea-970b-666b88b155f0.png)


to become below on screen widths under 600px


![image](https://user-images.githubusercontent.com/1094999/93794817-994a2d00-fc06-11ea-9878-084a78f3fe02.png)

Reply:
other css in `src/applications/disability-benefits/view-payments/sass/responsive-table.scss` was specific to `src/applications/disability-benefits/view-payments/`'s needs

Reply:
Ah okay...Would you mind communicating this on that design system ticket, that we are now managing separate implementations across apps? My team is going to be working on a responsive table component here soon too.

Reply:
See https://github.com/department-of-veterans-affairs/vets-design-system-documentation/issues/203#issuecomment-696254775